Modules/gcmodule.c

```diff
@@ -427,11 +507,7 @@ update_refs(PyGC_Head *containers)
          * permanent generation as the reference count is not accurately
```

Comment:
This comment is no longer valid and should be removed.

Reply:
Removed